scripts/utils/options.ts

```diff
@@ -0,0 +1,245 @@
+/**
+ * Use commander and prompts to gather a list of options for a script
+ */
+
+import prompts from 'prompts';
+import type { PromptObject } from 'prompts';
+import program from 'commander';
+import type { Command } from 'commander';
+import kebabCase from 'lodash/kebabCase';
+
+// Option types
+
+export type OptionId = string;
+export type BaseOption = {
+  description?: string;
+  /**
+   * By default the one-char version of the option key will be used as short flag. Override here,
```

Comment:
What if there are multiple options starting with the same char like `save` and `skip`?

Reply:
I guess that's when you override it?

Reply:
Do you have a different idea?